File: Python/8py 2/quarter_of.py
"""
|--------------------------------------------------------------------------
| Quarter of the year
|--------------------------------------------------------------------------
|
"""

def quarter_of(month):
   return 1 if month<=3 else 2 if month<=6 else 3 if month<=9 else 4


def quarter_of(month):
   return (month + 2) // 3


def quarter_of(month):
   return (month-1) // 3 + 1 


from math import ceil
def quarter_of(month):
   return ceil(month / 3)


print(quarter_of(3), 1)
print(quarter_of(8), 3)
print(quarter_of(11), 4)

File: JavaScript/8js 2/quarterOf.js
/*
|--------------------------------------------------------------------------
| Quarter of the year
|--------------------------------------------------------------------------
|
*/

function quarterOf(month) {
   return month <= 3 ? 1 : month <= 6 ? 2 : month <= 9 ? 3 : 4;
}


function quarterOf(month) {
   return Math.ceil(month / 3);
}


function quarterOf(month) {
   return (month <= 3 && 1) || (month <= 6 && 2) || (month <= 9 && 3) || (month <= 12 && 4);
}


console.log(quarterOf(3), 1);
console.log(quarterOf(8), 3);
console.log(quarterOf(11), 4);